src/abstractdataframe/sort.jl

```diff
@@ -333,31 +351,45 @@ Sort.defalg(df::AbstractDataFrame, o::Ordering; alg=nothing, cols=[]) =
 ## Actual sort functions
 ########################
 
+const SORT_ARGUMENTS =
 """
-    issorted(df::AbstractDataFrame, cols=All();
-             lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
+`cols` can be any column selector ($COLUMNINDEX_STR; $MULTICOLUMNINDEX_STR). If
+`cols` selects no columns, check whether `df` is sorted on all columns (this
+behaviour is deprecated and will change in future versions).
```

Comment:
This text (that we just added) needs to be adapted a bit depending on the function.

Reply:
right - fixed